ui/pages/Transaction.tsx

```diff
@@ -113,7 +113,24 @@ const TransactionPageContent = () => {
     if (isCustomAppError(error)) {
       throwOnResourceLoadError({ resource: 'tx', error, isError: true });
     }
-  }
+  } else {
+          // Display a specific failure message for failed transactions
+          return (
+            <div>
+              <TextAd mb={ 6 }/>
+              <PageTitle
+                title="Transaction details"
+                backLink={ backLink }
+                contentAfter={ tags }
+                secondRow={ titleSecondRow }
+              />
+              <p style={{ color: 'red', fontWeight: 'bold', marginTop: '20px' }}>
+                Address failed to call method on address.
```

Comment:
Hi! It's not exactly what the issue is about.
The idea was to implement a stub for transaction with a failed status, not an error state of the page.
Also, it has to be not the plain text, but addresses and method name from the transaction data.
And we use chakra-ui and variables for colors and paddings, so styling is not correct also